pkg/diagnostics/errorcode_monitoring.go

```diff
@@ -0,0 +1,52 @@
+package diagnostics
+
+import (
+	"context"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+
+	diagUtils "github.com/dapr/dapr/pkg/diagnostics/utils"
+	"github.com/dapr/dapr/pkg/messages/errorcodes"
+)
+
+type errorCodeMetrics struct {
+	errorCodeCount *stats.Int64Measure
+
+	appID   string
+	ctx     context.Context
+	enabled bool
+}
+
+func newErrorCodeMetrics() *errorCodeMetrics {
+	return &errorCodeMetrics{ //nolint:exhaustruct
+		errorCodeCount: stats.Int64(
+			"error_code/count",
+			"Number of times an error with a specific errorcode was encountered.",
+			stats.UnitDimensionless),
+
+		ctx:     context.Background(),
+		enabled: false,
```

Comment:
AppID?

Reply:
app-id is supplied with the `Init()` function, following the other metrics' structures
https://github.com/jake-engelberg/dapr//blob/5c3b42bf03519008739c1fc2ec8030dba5f1fd00/pkg/diagnostics/errorcode_monitoring.go#L48